### PR TITLE
MDAnalysis documentation style

### PR DESCRIPTION
**Pull request by ianmkenney**
Applied the MDAnalysis documentation style

### CODE REVIEW

**Comment by ianmkenney** [comment]
@orbeckst The docs workflow uploads an artifact of the current build. You can download it from within the actions panel.

File: mdaencore/similarity.py [hunk 1]
```diff
@@ -58,7 +58,6 @@
 
 .. bibliography::
     :filter: False
-    :style: MDA
```
Comment (orbeckst) [inline mdaencore/similarity.py:61]: not shipping the MDA bib style?

File: setup.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 
 
 def extensions(debug=False, use_cython=True):
+    # TODO: remove fast-math?
```
Comment (orbeckst) [inline setup.py:30]: yes, but that's better done in a separate PR — I totally agree with adding the comment here

**Review by orbeckst — CHANGES_REQUESTED** [review]
Overall looks good but we also need the standard footer with notes on privacy for anything served under mdanalysis.org.

**Comments**

pr_docs/autosummary/mdaencore.html contains the tests, which should be excluded. The overview table should also have text entries for each module although I assume that's for a later PR when you actually look at docs.

pr_docs/autosummary/mdaencore.bootstrap.html shows functions but they are not active links. In general there are issues with functions/methods not resolving. Presumably that can be fixed later, given that this PR is about the style.

The module docs contain "Author" and "new in version 0.16.0" entries that should be removed/consolidated.

File: mdaencore/similarity.py (hunk 1 above)
Comment (ianmkenney) [reply mdaencore/similarity.py:61]: This wasn't done in the hole2 kit docs either, trying to keep it consistent between the two packages for when we create a proper extension

Comment (orbeckst) [reply mdaencore/similarity.py:61]: ok, references in the hole kit look reasonable (although not as nice in the MDA docs) so I am fine with keeping it simple for right now. Maybe something for a docs extension.

**Comment by orbeckst** [comment]
Is there a RTD doc build on PR that allows one to preview the docs?

**Comment by ianmkenney** [comment]
The docs are available as build artifacts in [actions](https://github.com/MDAnalysis/mdaencore/actions/runs/5943121014).

**Comment by orbeckst** [comment]
Yeah, but that's so inconvenient that no reviewer will waste time on it. ... so #9 ;-)
